core/lib/snt/core/open_tracing/gem_ext.rb

```diff
@@ -42,4 +45,11 @@ def self.current_trace_object
       end
     end
   end
+  
+  # We need to make sure that Global Tracer for OpenTracing is set to ElasticAPM Tracer. This is basically called in the beginning
+  # of all methods extended in this module. The reason we set the global tracer here is because for some unknown reasons (possibly process forking)
+  # the global tracer is not set when we call into the extended methods in this module even when we set it in application.rb or initializers in IFC or PMS
+  def self.set_global_tracer_to_elastic
+    ::OpenTracing.global_tracer = ElasticAPM::OpenTracing::Tracer.new unless ::OpenTracing.global_tracer.class == ElasticAPM::OpenTracing::Tracer
```

Comment:
this is a little more ruby like, if you wanted...
`::OpenTracing.global_tracer = ElasticAPM::OpenTracing::Tracer.new unless ::OpenTracing.global_tracer.instance_of?(::ElasticAPM::OpenTracing::Tracer)
`

Reply:
Can you fix the two Rubocop flags on this file?